components/timeline/TimelineDiscover.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import type { Recommendation } from '../composables/recommendations'
+import { getLanguageForRecs } from '../../utils/language'
+
+const { locale: lang } = useI18n()
+const locale = getLanguageForRecs(lang.value)
+
+const recommendations: Recommendation[] = await $fetch(`/api/recommendations?locale=${locale}`)
+</script>
+
+<template>
+  <div>
+    Coming soon...
+  </div>
+</template>
```

Comment:
Imagine this gif on the page...
![under construction](https://github.com/MozillaSocial/elk/assets/1273879/73e6b0ab-4cb2-4c1b-9eb2-cdd45439357f)